app/src/main/java/org/commcare/commcaresupportlibrary/identity/model/Verification.java

```diff
@@ -0,0 +1,90 @@
+package org.commcare.commcaresupportlibrary.identity.model;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+
+@SuppressWarnings({"unused"})
+public class Verification implements Parcelable {
+
+    private int confidence;
+    private Tier tier;
+    private String guid;
+
+
+    /**
+     * Empty Constructor
+     */
+    public Verification() {
+
+    }
+
+    /**
+     * This constructor creates a new verification
+     *
+     * @param confidence An int containing the (matching) confidence
+     * @param tier       The tier score derived from the confidence
+     * @param guid       Global unique id of the verified person
+     */
+    public Verification(int confidence, @NonNull Tier tier, @NonNull String guid) {
```

Comment:
These are the "result" objects from the Identify Provider, correct? It might help to call them, like, "VerificationMatch" to clarify their intent. 

Along the same lines:
* I'd also make sure it's clear whether the GUID is the Identity System guid or whether it's the CommCare ID. 
* I'd call "tier" "Strength" or "MatchStrength" (depending on the object name).